global/meta/models.yml

```diff
@@ -644,6 +650,17 @@ committee:
     restriction_mode: A
     read_only: true
     description: "Calculated field."
+
+  user_$_management_level:
+    type: template
+    restriction_mode: B
+    replacement_enum:
+      - can_manage
```

Comment:
This field was strange before but it now even stranger.

The reason of template fields is, that there is a variable number of replacement values that create field names that are only known at run-time.

In this case, this is always only one field `user_$can_manage_management_level`. Since there is no other possible field, there is not even a need to look into the template field.

Would it be possible for you to replace this field by a normal field like

```yml
manager_ids:
  type: relation-list
  to: user/committee_manager_ids
description: List of users ids, that have the committee management permission
```

This would make the code much easier to read

Reply:
This field is a new field and we need it to get all related users in case of deleting a committee. It's absence was the reason that committee.delete left the database in a corrupt state, where it was impossible to create a new committee.
The (now!) related field  user.commitee_$_management_level was incorparated with the idea, that there will be more than one permission, a user could have for a specific committee and the opposite relation has to reflect this.

Reply:
I still don't understand why a template field is needed with only one possible replacement. Why can't you use a normal relation field?